src/test/java/io/xpring/xrpl/FakeXRPProtobufs.java

```diff
@@ -27,6 +27,32 @@
     static long testDrops = 10;
 
     static int testDestinationTag = 2;
+
+    static ByteString memoDataBytes;
+    static ByteString memoFormatBytes;
+    static ByteString memoTypeBytes;
+
+    static {
+        try {
+            memoDataBytes = ByteString.copyFrom("123", "Utf8");
+        } catch (UnsupportedEncodingException e) {
```

Comment:
don't use one letter variable names. Saves time now, difficult to read later :) 

(Yes, I know it's only test code but it's good to have good habits)

Reply:
unlearning bad habits they taught me at school